### PR TITLE
chore: discard the deprecation of LsBoot, LsDev and LsSysFirmware

### PR DESCRIPTION
**Pull request by xiangce**
- They are used by gss rules and other apps
   See the latest comment to #3833

### All Pull Requests:

Check all that apply:

* [x] Have you followed the guidelines in our Contributing document, including the instructions about commit messages?
* [ ] Is this PR to correct an issue?
* [ ] Is this PR an enhancement?

### CODE REVIEW

File: insights/parsers/ls_boot.py [hunk 1]
```diff
@@ -38,20 +38,19 @@
     True
 """
 
-from .. import FileListing, parser, CommandParser
+from insights import CommandParser, FileListing, parser
```
Comment (ryan-blakley) [inline insights/parsers/ls_boot.py:41]: @xiangce Can you use the new FileListing [function](https://github.com/RedHatInsights/insights-core/blob/master/insights/parsers/ls.py#L44) that was added for these parsers? If not the deprecation would need to also be removed from the old FileListing [function](https://github.com/RedHatInsights/insights-core/blob/master/insights/core/__init__.py#L1644).

Comment (xiangce) [reply insights/parsers/ls_boot.py]: @ryan-blakley - Yeap, I missed this. Fixed in the new commit.

Thanks for reviewing.

**Comment by xiangce** [comment]
test me

**Comment by xiangce** [comment]
test me

**Review by ryan-blakley — APPROVED** [review]

**Pull request merged**